frontend/src/routes/WildcardRoute.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Navigate, useSearchParams } from 'react-router-dom-v5-compat';
+
+import * as AlertUtils from '../utils/AlertUtils';
+
+export const WildcardRoute = (): JSX.Element => {
+  const [searchParams] = useSearchParams();
+  const openshiftError = searchParams.get('openshift_error');
+
+  if (openshiftError) {
+    AlertUtils.addError(
+      'Openshift authentication to remote cluster failed',
```

Comment:
I'm not sure I would call this a "remote" cluster although it is remote to Kiali itself. It would be nice if we could put the cluster name in the alert message.

Reply:
I guess is a remote cluster, right, or is there any other case?. I will add the cluster name.

Reply:
> I guess is a remote cluster, right, or is there any other case?. I will add the cluster name.

It's "remote" to where Kiali is installed but in terms of Istio deployments it could be another primary in a multi-primary deployment so I'm worried that calling it "remote" could confuse people.